### PR TITLE
Allow people to use another user that has root privileges when installing centreon

### PR DESCRIPTION
**Pull request by tanguyvda**
in some cases the root user is not usable. 
Not sure about the variables naming standards (DEFAULT_root_user or DEFAULT_ROOT_USER) and the label can perhaps be more understandable

i've set $root_user = "" in functions.php while i've made it mandatory to fill the field in step6.php. Not the smartest move...

### CODE REVIEW

File: www/install/steps/step6.php [hunk 1]
```diff
@@ -38,6 +38,7 @@
 DEFINE('DEFAULT_CONF_NAME', 'centreon');
 DEFINE('DEFAULT_STORAGE_NAME', 'centreon_storage');
 DEFINE('DEFAULT_UTILS_NAME', 'centreon_status');
+DEFINE('DEFAULT_root_user', 'root');
```
Comment (sc979) [inline www/install/steps/step6.php]: DEFAULT_root_user => DEFAULT_ROOT_USER

File: www/install/steps/step6.php [hunk 2]
```diff
@@ -85,7 +87,14 @@
             </td>
         </tr>
         <tr>
-            <td class='formlabel'>"._('Root password')."</td>
+            <td class='formlabel'>"._('User with root privileges').$star."</td>
```
Comment (sc979) [inline www/install/steps/step6.php]: Did you add : _('User with root privileges')  to the gettext file ? 
Please add the gettext reference file too

File: www/install/steps/step6.php [hunk 3]
```diff
@@ -85,7 +87,14 @@
...
+            <td class='formvalue'>
+                <input type='text' name='root_user' value='".$defaults['root_user']."' />
+                <label class='field_msg'></label>
+            </td>
+        </tr>
+        <tr>
+            <td class='formlabel'>"._('User with root privileges password')."</td>
```
Comment (sc979) [inline www/install/steps/step6.php]: Did you add : _('User with root privileges password')  to the gettext file ? 
Please add the gettext reference file too

**Comment by cgagnaire** [comment]
Replaced by https://github.com/centreon/centreon/pull/7445.

**Pull request closed without merging**